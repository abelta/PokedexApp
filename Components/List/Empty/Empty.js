import React from 'react';
import { Text, View } from 'react-native';

const Empty = () => (
  <View>
    <Text>...loading data</Text>
  </View>
);

export default Empty;
