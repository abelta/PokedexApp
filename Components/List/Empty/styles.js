import { StyleSheet } from 'react-native';

const styles = StyleSheet.create({
  empty: {
    borderColor: 'lightgrey',
    borderWidth: StyleSheet.hairlineWidth,
    height: 35,
    justifyContent: 'center',
  },
  text: {
    fontSize: 18,
    textAlign: 'center',
  },
});

export default styles;
