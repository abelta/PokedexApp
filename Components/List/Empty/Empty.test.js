import React from 'react';
import renderer from 'react-test-renderer';
import Empty from '.';

describe('Empty', () => {
  const tree = renderer.create(<Empty />).toJSON();
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
