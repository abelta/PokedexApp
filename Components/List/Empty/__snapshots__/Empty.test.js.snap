// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "borderColor": "lightgrey",
        "borderWidth": 0.5,
        "height": 35,
        "justifyContent": "center",
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 18,
        "textAlign": "center",
      }
    }
  >
    ...loading data
  </Text>
</View>
`;
